suite/tests/api/dis-x64.expect

```diff
@@ -308,11 +308,12 @@
 +0x02d3   63 15 16 c2 b9 d4    movsxd edx, <rel> dword ptr [0xffffffffe4b9c4ef]
 +0x02d9   d2 79 7d             sar    byte ptr [rcx+0x7d], cl
 +0x02dc   cb                   retf
-+0x02dd   c7 7c bb e4 8e 81 93 xbegin 0x0000000054938473
++0x02dd   c7 7c bb e4 8e 81 93...??  <INVALID>
  44
-+0x02e5   7f e2                jnle   0x00000000100002c9
-+0x02e7   e1 66                loope  0x000000001000034f
-+0x02e9   c0 ea 87             shr    dl, 0x87
++0x02de   7c bb                jl     0x000000001000029b
```

Comment:
> I will say that I don't really understand the changes in the dis test. Are those code files randomly generated?

I assume you clobbered these expect files with the new output, right?
So I'm not sure what you're asking?  The binary file being decoded is fixed.
0x2dd is now invalid; so it tries again one byte after the start of the invalid, which is why these couple of lines also changed, until the offset lines up again.

If you mean where did the binary file come from: from a udis86 test.  See suite/tests/api/dis-udis86-randtest.license

Reply:
Yeah, I meant the instruction stream doesn't seem to make any sense. If it's randomly generated that explains a lot.